src/Solucion_JavierRamiro/src/programacion/empresaprogramacion/Gerente.java

```diff
@@ -0,0 +1,23 @@
+package programacion.empresaprogramacion;
+
+public class Gerente extends Empleado {
+  private String departamento;
+
+  public Gerente(String nombre, double sueldo, String departamento2) {
+      super(nombre, sueldo);
```

Comment:
Inicializa el atributo departamento también, no???

```suggestion
      super(nombre, sueldo);
      departamento = departamento2;
```